src/main/java/Window/Controller.java

```diff
@@ -1,22 +1,75 @@
-package Window;
+package window;
 
-import javafx.application.Application;
 import javafx.application.Platform;
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.scene.control.TextArea;
 import javafx.scene.input.MouseEvent;
-import javafx.scene.layout.Pane;
+import javafx.scene.control.Button;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.control.MenuBar;
+import javafx.scene.control.MenuItem;
 
+import java.io.IOException;
+import java.io.File;
+import javafx.stage.FileChooser;
 import java.text.SimpleDateFormat;
 
 /**
  * Created by 101010 on 8-5-2017.
  */
 public class Controller {
+    @FXML private Button browse;
+
+    @FXML private MenuBar menu;
+
+    @FXML private MenuItem openFile;
+
     @FXML private TextArea console;
 
+    @FXML private void open(final ActionEvent event) throws IOException {
```

Comment:
I seems like this method could handle multiple types of opening request. I suggest you specialize this method to only be invoked when openFile is clicked.